examples/TrustPilotWidget/src/TrustPilot.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const TrustBox = ({businessUnitId= "5ddd3f26dcc430000159acff", templateId= "5419b6a8b0d04a076446a9ad"}) => {
+    // Create a reference to the <div> element which will represent the TrustBox
+    const ref = React.useRef(null);
+    React.useEffect(() => {
+        // If window.Trustpilot is available it means that we need to load the TrustBox from our ref.
+        // If it's not, it means the script you pasted into <head /> isn't loaded  just yet.
+        // When it is, it will automatically load the TrustBox.
+        if (window.Trustpilot) {
+            window.Trustpilot.loadFromElement(ref.current, true);
+        }
+    }, []);
```

Comment:
should businessUnitId and templateId be added to the useEffect dependencies ? 

`}, [businessUnitId, templateId]);`

This way the trustpilot component will re-render when businessUnitId or templateId are changed in the settings and the change will be visible instantly without needing to save&reload the page.